### PR TITLE
Option for disable global ajax events

### PR DESCRIPTION
**Pull request by VoroninWD**
PR: https://github.com/octobercms/october/pull/4282

### CODE REVIEW

File: ajax-attributes-api.md [hunk 1]
```diff
@@ -15,6 +15,7 @@ Attribute | Description
 **data-request-redirect** | specifies a URL to redirect the browser after the successful AJAX request.
 **data-request-url** | specifies a URL to which the request is sent. default: `window.location.href`
 **data-request-update** | specifies a list of partials and page elements (CSS selectors) to update. The format is as follows: `partial: selector, partial: selector`. Usage of quotes is required in some cases, for example: `'my-partial': '#myelement'`. If the selector string is prepended with the `@` symbol, the content received from the server will be appended to the element, instead of replacing the existing content. If the selector string is prepended with the `^` symbol, the content will be prepended instead.
+**data-request-ajax-global** | ``false`` by default. Set ``true`` for enable default jQuery [ajax events](http://api.jquery.com/category/ajax/global-ajax-event-handlers/): ``ajaxStart``, ``ajaxStop``, ``ajaxComplete``, ``ajaxError``, ``ajaxSuccess`` and ``ajaxSend``.
```
Comment (w20k) [inline ajax-attributes-api.md]: @VoroninWD  could you make a change: `Set ``true`` for enable default jQuery`
to `Set ``true`` to enable jQuery ajax events globally`

File: ajax-attributes-api.md [hunk 2]
```diff
@@ -15,6 +15,7 @@ Attribute | Description
 **data-request-redirect** | specifies a URL to redirect the browser after the successful AJAX request.
 **data-request-url** | specifies a URL to which the request is sent. default: `window.location.href`
 **data-request-update** | specifies a list of partials and page elements (CSS selectors) to update. The format is as follows: `partial: selector, partial: selector`. Usage of quotes is required in some cases, for example: `'my-partial': '#myelement'`. If the selector string is prepended with the `@` symbol, the content received from the server will be appended to the element, instead of replacing the existing content. If the selector string is prepended with the `^` symbol, the content will be prepended instead.
+**data-request-ajax-global** | ``false`` by default. Set ``true`` to enable jQuery [ajax events](http://api.jquery.com/category/ajax/global-ajax-event-handlers/) globally : ``ajaxStart``, ``ajaxStop``, ``ajaxComplete``, ``ajaxError``, ``ajaxSuccess`` and ``ajaxSend``.
```
Comment (LukeTowers) [inline ajax-attributes-api.md]: What's with all the double backticks?

Comment (w20k) [reply ajax-attributes-api.md]: @VoroninWD could you remove double backticks.

Comment (LukeTowers) [reply ajax-attributes-api.md]: @VoroninWD They should be single backticks, my apologies for not being clearer.

**Pull request merged**